Create destination docs

rsuski-acxiom opened the pull request.

rsuski-acxiom committed:
Add files via upload

rsuski-acxiom committed:
Create test

rsuski-acxiom committed:
Add files via upload

rsuski-acxiom committed:
Delete help/destinations/assets/catalog/advertising/acxiom-real-id-audience-connection/test

rsuski-acxiom committed:
Add files via upload

LRUGGE committed:
Update acxiom-audience-connection.md

changed product name under supported destinations section to the new name.

LRUGGE committed:
Update acxiom-real-id-audience-connection.md

updated the name referenced under support destinations to the new name for this tile. Also added to the description for the real id field under mapping section

LRUGGE committed:
Update acxiom-real-id-audience-connection.md

changed a couple of words in the mapping area to singular instead of plural since there is only one field to map.

LRUGGE committed:
Update acxiom-real-id-audience-connection.md

Removed a sentence from the mapping section.

rsuski-acxiom committed:
Delete help/destinations/catalog/advertising/acxiom-audience-distribution.md

replaced with acxiom-audience-connection

rsuski-acxiom committed:
Update acxiom-real-id-audience-connection.md

LRUGGE committed:
Update acxiom-real-id-audience-connection.md

added two new sentences to the doc. one under use cases and a third bullet under prereq's

vfalon commented on help/destinations/catalog/advertising/acxiom-real-id-audience-connection.md:
```diff
@@ -0,0 +1,123 @@
+
+## Use cases {#use-cases}
+
+This connector supports clients who have Acxiom Real Identity loaded into Real-Time CDP (RTCDP) as an identifier. To help you better understand how and when you should use the [!DNL Acxiom Real ID Audience Connection] destination, here is a sample use case that [!DNL Adobe Experience Platform] customers can solve by using this connector.
```
Remove all mentions of RTCDP. The correct public-facing terminology is Real-Time CDP.

LRUGGE replied:
removed RTCDP

vfalon commented on help/destinations/catalog/advertising/acxiom-real-id-audience-connection.md:
```diff
@@ -0,0 +1,123 @@
+## Prerequisites {#prerequisites}
+- **Confirm Terms of Use:** Before you can configure a new [!DNL Acxiom Real ID Audience Connection] destination, you must read and sign [!DNL Acxiom’s] Terms of Use Agreement. You will receive the link to the agreement once your executed sales order is complete. Until you sign the agreement, you will not see the [!DNL Acxiom Real ID Audience Connection] destination card in the Experience Platfor ...
+- **Know your Adobe organization ID:** Your [!DNL Adobe] organization ID is needed to complete your Terms of User Agreement. See [!DNL Adobe's] *Organizations in Experience Cloud* topic for details on how to [view your organization ID](https://experienceleague.adobe.com/en/docs/core-services/interface/administration/organizations#concept_EA8AEE5B02CF46ACBDAD6A8508646255).
+- **Obtain License for [DNL! Acxiom's Real ID] product:** Once a license is obtained, make Acxiom's Real ID available within RTCDP. See [Acxiom Data Enhancement](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/catalog/data-partner/acxiom-data-enhancement) for more information.
```
lowercase `l` in `License`

LRUGGE replied:
corrected

vfalon commented on help/destinations/catalog/advertising/acxiom-real-id-audience-connection.md:
```diff
@@ -0,0 +1,123 @@
+- **Obtain License for [DNL! Acxiom's Real ID] product:** Once a license is obtained, make Acxiom's Real ID available within RTCDP. See [Acxiom Data Enhancement](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/catalog/data-partner/acxiom-data-enhancement) for more information.
+
+
+## Supported Destinations {#supported-destinations}
```
`## Supported destinations {#supported-destinations}`

LRUGGE replied:
corrected

vfalon commented on help/destinations/catalog/advertising/acxiom-real-id-audience-connection.md:
```diff
@@ -0,0 +1,123 @@
+
+## Supported Destinations {#supported-destinations}
+
+[!DNL Acxiom’s Real ID Audience Connection] destination currently supports audience activation to the following platforms.<br> 
```
Remove the `<br> `

LRUGGE replied:
removed

vfalon commented on help/destinations/catalog/advertising/acxiom-real-id-audience-connection.md:
```diff
@@ -0,0 +1,123 @@
+>- To activate data, you need the **View Destinations**, **Activate Destinations**, **View Profiles**, and **View Segments** ![access control permissions](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/home#permissions). Read the ![access control overview](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/ui/overview) or contact your prod ...
+>- To export identities, you need the **View Identity Graph** ![access control permission](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/home#permissions).
+
+<br>
```
Remove `<br>` tags from the page. No need for these.

LRUGGE replied:
Removed

vfalon commented on help/destinations/catalog/advertising/acxiom-real-id-audience-connection.md:
```diff
@@ -0,0 +1,123 @@
+
+[!DNL Acxiom Real ID Audience Connection] only allows mapping to the following target field.  
+
+| Field Name | Description|Required | Field Order |
```
What is the point of the Field Order column here? There's only one field that we are mapping.

LRUGGE replied:
Removed.

vfalon commented on help/destinations/catalog/advertising/acxiom-real-id-audience-connection.md:
```diff
@@ -0,0 +1,123 @@
+In the **[!UICONTROL Source Field]** column, enter the name of the source attribute that you want to map to the corresponding target field, or select the arrow icon to open the **[!UICONTROL  Select source field]** screen.<br>
+![Mapping screen](../../assets/catalog/advertising/acxiom-real-id-audience-connection/real_id_mapping_screen.png)
+
+After you have mapped all the fields, select **[!UICONTROL Next]**.
```
"After you have mapped all the fields, select **[!UICONTROL Next]**."

I'm not sure this phrase applies here. Isn't there only one mapping supported for this destination?

LRUGGE replied:
Removed "After you have mapped all the fields,"

vfalon commented on help/destinations/catalog/advertising/acxiom-real-id-audience-connection.md:
```diff
@@ -0,0 +1,123 @@
+- **Obtain License for [DNL! Acxiom's Real ID] product:** Once a license is obtained, make Acxiom's Real ID available within RTCDP. See [Acxiom Data Enhancement](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/catalog/data-partner/acxiom-data-enhancement) for more information.
+
+
+## Supported Destinations {#supported-destinations}
```
After the prerequisites section we need to include the two missing sections as shown in the docs template: https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.

`Suported identities` - add information about the Real ID like you see in the table in the template.
`Supported audiences` - include all audience types supported by the destination

LRUGGE replied:
Pending. Getting more info. from our team. Stay tuned.

LRUGGE replied:
This is complete now. The two additional sections have been added.

vfalon commented on help/destinations/catalog/advertising/acxiom-real-id-audience-connection.md:
```diff
@@ -0,0 +1,123 @@
+- Exported file name
+
+
+## Next steps {#next-steps}
```
Move the Troubleshooting and Next steps sections to the end of the page (after data usage).

LRUGGE replied:
Done.

vfalon reviewed (changes requested):
@rsuski-acxiom Thanks for submitting the docs updates! There are a few changes to make before we get this rolling. See my comments please and let us know when the updates are ready.

LRUGGE committed:
Update acxiom-real-id-audience-connection.md

Made adjustments based on feedback from Adobe.

LRUGGE committed:
Update acxiom-real-id-audience-connection.md

Added the following two sections per Adobe's request based on their template. These sections may have to be edited again once I hear back from product.
-Supported identities
-Supported audiences
